TombEngine/Game/Lara/lara_climb.cpp

```diff
@@ -344,8 +347,14 @@ void lara_col_climb_idle(ItemInfo* item, CollisionInfo* coll)
 			}
 			else
 			{
-				item->Animation.TargetState = LS_LADDER_TO_CROUCH;
-				item->Animation.RequiredState = LS_CROUCH_IDLE;
+				// Hack: disable this state transition to avoid clipping bug on issue 1584.
```

Comment:
I am not sure how this change fixes ladder to crouch issue, if it completely disables such state change? Does `lara_col_climb_up` automatically sort it out?